packages/headless/src/api/commerce/product-listings/product-listing-params.ts

```diff
@@ -1,7 +1,39 @@
-import {HTTPContentType, HttpMethods} from '../../platform-client';
+import {FacetOptions} from '../../../features/facet-options/facet-options';
+import {AnyFacetRequest} from '../../../features/facets/generic/interfaces/generic-facet-request';
+import {
+  HTTPContentType,
+  HttpMethods,
+  PlatformClientCallOptions,
+} from '../../platform-client';
 import {BaseParam} from '../../platform-service-params';
+export type ProductListingsParam = ProductListingBaseParam &
+  ProductListingRequestParam;
 
-export type ProductListingsParam = BaseParam;
+export type ProductListingBaseParam = Omit<BaseParam, 'url'> & {
```

Comment:
I'm cautious of `Omit` because if `BaseParam` gets another property, then it will either get into this interface, or the dev would need to maintain the list of strings. Could it be simpler to duplicate `accessToken` and `organizationId`?